base/12-alpine/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:12-alpine
+
+RUN apk update && \
+  apk add \
```

Comment:
I basically went through `apk`'s unsatisfiable constraints and looked for package contents on https://pkgs.alpinelinux.org/packages to find the files.